server/src/main/java/com/hedera/block/server/Server.java

```diff
@@ -82,16 +85,33 @@ public static void main(final String[] args) {
             final GrpcRouting.Builder grpcRouting =
                     GrpcRouting.builder().service(blockStreamService);
 
+            @NonNull final HealthService healthService = new HealthServiceImpl(serviceStatus);
+
+            @NonNull
+            final HttpRouting.Builder httpRouting =
+                    HttpRouting.builder()
+                            .register(healthService.getHealthRootPath(), healthService);
+
             // Build the web server
+            // TODO: make port server a configurable value.
             @NonNull
             final WebServer webServer =
-                    WebServer.builder().port(8080).addRouting(grpcRouting).build();
+                    WebServer.builder()
+                            .port(8080)
+                            .addRouting(grpcRouting)
+                            .addRouting(httpRouting)
+                            .build();
 
             // Update the serviceStatus with the web server
             serviceStatus.setWebServer(webServer);
 
             // Start the web server
             webServer.start();
+
+            // Log the server status
+            LOGGER.log(
+                    System.Logger.Level.INFO,
+                    "Block Node Server started at port: " + webServer.port());
```

Comment:
Maybe we should change this log output to: `Block Node Server started on port(s): `? I imagine we'll eventually have services on different ports (gRPC, REST, GraphQL, etc)

Reply:
maybe in the future we can change the message or create different messages for each port exposed.

With this we have both http and grpc on same port.